### PR TITLE
Acknowledge PS 8.2.x

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Acknowledge PS 8.2.x
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Sponsor company   | 
| How to test?      | CI should be green

### CODE REVIEW

File: src/PullRequest/Domain/Aggregate/PullRequest/PullRequest.php [hunk 1]
```diff
@@ -101,7 +101,8 @@ public function addLabelsByDescription(PullRequestDescription $description): voi
         // Remove some labels in labels list
         $this->labels = array_diff($this->labels, [
             'develop',
-            '8.1.x',
+            '8.2.x',
+            '8.2.x',
```
Comment (nicosomb) [inline src/PullRequest/Domain/Aggregate/PullRequest/PullRequest.php:105]: are you sure to duplicate `8.2.x`?

File: src/PullRequest/Domain/Aggregate/PullRequest/PullRequestDescription.php [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 class PullRequestDescription
 {
-    public const TARGET_BRANCH_AVAILABLE = ['develop', '8.1.x'];
+    public const TARGET_BRANCH_AVAILABLE = ['develop', '8.2.x', '8.2.x'];
```
Comment (nicosomb) [inline src/PullRequest/Domain/Aggregate/PullRequest/PullRequestDescription.php:11]: are you sure to duplicate `8.2.x`?

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: src/PullRequest/Domain/Aggregate/PullRequest/PullRequestDescription.php (hunk 2 above)
Comment (matks) [reply src/PullRequest/Domain/Aggregate/PullRequest/PullRequestDescription.php:11]: NO. This is what I get for trying to fix CI when tired at 10pm 😅

**Review by nicosomb — APPROVED** [review]

**Comment by matks** [comment]
CI is failing to deploy on integration environment

I wonder if this is because the PR comes from my fork (see https://michaelheap.com/access-secrets-from-forks/) so I try with a PR which does not come from a fork https://github.com/PrestaShop/kanbanbot/pull/54

**Comment by matks** [comment]
It was the fork indeed https://github.com/PrestaShop/kanbanbot/pull/54

**Pull request closed without merging**